github/Organization.pyi

```diff
@@ -194,6 +195,7 @@ class Organization(CompletableGithubObject):
     def remove_from_membership(self, member: NamedUser) -> None: ...
     def remove_from_public_members(self, public_member: NamedUser) -> None: ...
     def remove_outside_collaborator(self, collaborator: NamedUser) -> None: ...
+    def create_project(self, param: str, body: github.GithubObject.NotSet) -> None: ...
```

Comment:
This should follow the existing pattern of using _NotSetType, so you can remove that import.